test/integration/crud/explain.test.ts

```diff
@@ -117,6 +119,111 @@ describe('CRUD API explain option', function () {
       });
     }
   }
+
+  describe('explain helpers w/ maxTimeMS', function () {
```

Comment:
I think in order to capture intentionality, whichever way the spec lands, we'll want to add cases for when maxTimeMS is specified as a command level option but not an explain-level option.

It may be easier to enumerate all these test cases if we use describe blocks for `describe('when maxTimeMS <is/is not> set as an explain option')` with `it('<what happens when it is/isn't set on the command-level>')` (or the reverse, with the is/is not set as command-level option in the `describe` and the explain in the `it`), we could also use nested describes to fully separate the `when` from the `what happens`, but I don't know whether that would help or hurt the readability.